ydb/tests/library/harness/kikimr_config.py

```diff
@@ -432,7 +432,32 @@ def __init__(
 
             self.yaml_config["feature_flags"]["enable_external_data_sources"] = True
             self.yaml_config["feature_flags"]["enable_script_execution_operations"] = True
-
+            
+        if os.getenv("YAML_CONFIG") is not None:      
```

Comment:
need prefix YDB_ - for prevent conflict with other systems, because environment variables are global for all processes.